index.html

```diff
@@ -1007,17 +944,6 @@ <h4>
         devices, such as writable tags, when they come in range, as
         <a>NDEF message</a>s.
       </li>
-      <li>
-        [future] Allow manual connection for various technologies such as
-        NFC-A and NFC-F depending on the secondary device.
-      </li>
-      <li>
-        [future] Allow <a>NFC handover</a> to Bluetooth or WiFi.
-      </li>
-      <li>
-        [future] Allow card emulation with secure element or host card
-        emulation.
-      </li>
     </ol>
```

Comment:
In general, I am thinking should we somehow keep these somewhere as informal content, to define the use case coverage of NFC in general (with possible future adoption into Web NFC).